docusaurus/docs/Android/05-ui-cookbook/15-watching-livestream.mdx

```diff
@@ -0,0 +1,360 @@
+---
+title: Watching a livestream
+description: How to watch a livestream on android with Kotlin
+---
+
+This cookbook tutorial walks you through how to render the UI for watching a livestream on Android.
+
+:::note
+This cookbook tutorial will skip how to join a livestream call. If you didn't learn about the ![Livestream Tutorial](../02-tutorials/03-livestream.mdx), we're highly recommend you to read about the tutorial first.
```

Comment:
`we're highly recommend` should be `we highly recommend`

Reply:
Nice catch!